crossdev

```diff
@@ -1623,9 +1649,11 @@ LLVM="${LLVM}" emerge-wrapper --target ${CTARGET} --init || exit 1
 ### but gcc relies on their presence by refering to =/lib64 as
 ### =/usr/lib/../lib64 when builds itself (see https://bugs.gentoo.org/652724)
 ###
-### Thus we create non-symlinked layout early.
-xmkdir -p "${EPREFIX}"/usr/${CTARGET}/lib
+### Thus we create the layout early.
 xmkdir -p "${EPREFIX}"/usr/${CTARGET}/usr/lib
+pushd "${EPREFIX}"/usr/${CTARGET} >/dev/null
+ln -s usr/lib lib
```

Comment:
See my other comment, but supposing we were to do that, we should do all the other symlinks too.